cfme/scripting/setup_env.py

```diff
@@ -14,14 +14,17 @@ def tot_time(string):
     mtch = re.match('^((?P<days>\d+)+d)?\s?((?P<hours>\d+)+h)?\s?((?P<minutes>\d+)+m)?\s?', string)
     tot = int(mtch.group('days') or 0) * 24 * 60
     tot += int(mtch.group('hours') or 0) * 60
-    tot += int(mtch.group('minutes')or 0)
+    tot += int(mtch.group('minutes') or 0)
     return tot
 
 
 def provision_appliances(count, cfme_version, provider, lease_time):
     sprout_client = SproutClient.from_config()
     apps, request_id = sprout_client.provision_appliances(version=str(cfme_version),
```

Comment:
**REQUIRED:** Though it's been used elsewhere I really dislike this formatting and would prefer
```python
    apps, request_id = sprout_client.provision_appliances(
        version=str(cfme_version), count=count, preconfigured=False,
        lease_time=lease_time, provider=provider
    )
```